lib/Interpreter/InterOp.cpp

```diff
@@ -2267,5 +2267,27 @@ namespace InterOp {
     }
     return "";
   }
+
+  bool IsConstMethod(TCppFunction_t method)
+  {
+
+    // auto *D = (clang::Decl *)method;
+    // auto *FD = llvm::dyn_cast_or_null<clang::FunctionDecl>(D);
+
+    // if (!FD->isCXXClassMember())
+    //   return false;
+      
+    // const auto func = dyn_cast<CXXMethodDecl>(FD);
+    // if (func && func->getMethodQualifiers().hasConst())
+    //   return true;
```

Comment:
Please delete the unused code. Also make all of this in one commit with a good commit message.